src/components/custom/signup.js

```diff
@@ -107,19 +102,18 @@ class Signup extends Component {
         }
     }
 
-    handleEmailSignup = (e) => {
+    const handleEmailSignup = (e) => {
         e.preventDefault()
-        this.setState({ is_submitting: true })
-        let { email, email_error_msg } = this.state
-        email = email.replace(/\s/g, '')
-        this.handleValidation(email)
-        const has_error_email = this.validateEmail(email)
-
+        setSubmitting(true)
+        // let { email, email_error_msg } = this.state
+        // email = email.replace(/\s/g, '')
+        handleValidation(email.replace(/\s/g, ''))
+        const has_error_email = validateEmail(email)
         if (has_error_email || email_error_msg) {
-            return this.setState({ is_submitting: false })
+            return setSubmitting('false')
```

Comment:
this should be a boolean instead of a string.

Reply:
Done